package.json

```diff
@@ -10,7 +10,8 @@
   },
   "devDependencies": {
     "eslint": "3",
-    "mocha": "3"
+    "mocha": "3",
+    "eslint-plugin-mocha": "4.9.0"
```

Comment:
@morrme ah. when we install via `npm i morrme/eslint-plugin-fxa#add-mocha-plugin` it should also install the dependency module